webapp/components/compat-2021.js

```diff
@@ -34,6 +39,218 @@ class Compat2021 extends PolymerElement {
   static get is() {
     return 'compat-2021';
   }
+
+  static get properties() {
+    return {
+      stable: Boolean,
+    };
+  }
+
+  ready() {
+    super.ready();
+
+    const params = (new URL(document.location)).searchParams;
+    this.stable = params.get('stable') !== null;
+  }
 }
 window.customElements.define(Compat2021.is, Compat2021);
 
+const STABLE_TITLES = [
+  'Chrome/Edge Stable',
+  'Firefox Stable',
+  'Safari Stable',
+];
+
+const EXPERIMENTAL_TITLES = [
+  'Chrome/Edge Dev',
+  'Firefox Nightly',
+  'Safari Tech Preview',
```

Comment:
```suggestion
  'Safari Technology Preview',
```

Or maybe "Safari Preview" or  "Safari TP" if it needs to be short.

Reply:
Went for "Safari Preview" for now as the full string was too long. We can always change it later.